src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,24 @@
+import { jwtService } from '../service/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  const userData = jwtService.verify(token);
```

Comment:
Consider wrapping `jwtService.verify(token)` in a try-catch block to handle any potential errors that might occur during token verification, such as malformed tokens.